core/client/src/main/java/alluxio/client/netty/NettyRemoteBlockReader.java

```diff
@@ -121,7 +121,7 @@ public ByteBuffer readRemoteBlock(InetSocketAddress address, long blockId, long
       }
       throw new IOException(e);
     } finally {
-      if (channel != null && listener != null) {
+      if (channel != null && listener != null && channel.isActive()) {
```

Comment:
I'd prefer not to crash the production server if some bug leads to this case (e.g. `NettyUnderFileSystemFileWriter` is invoked on the worker side). Does it make sense to print an error message instead in such case?